lib/sidekiq/cron/web_extension.rb

```diff
@@ -56,24 +56,24 @@ def self.registered(app)
 
         # Enqueue cron job.
         app.post '/cron/namespaces/:namespace/jobs/:name/enque' do
-          if job = Sidekiq::Cron::Job.find(route_params[:name], route_params[:namespace])
+          if (job = Sidekiq::Cron::Job.find(route_params[:name], route_params[:namespace]))
             job.enque!
           end
-          redirect params['redirect'] || "#{root_path}cron"
+          redirect params['redirect'] || "#{root_path}cron/namespaces/#{route_params[:namespace]}"
```

Comment:
Maybe we can create a helper to DRY this code? Something like "redirect_with_namespace".

Reply:
Should we implement the `redirect_with_namespace` method inline within the `WebExtension `like this?
```ruby
app.helpers do
  def redirect_with_namespace
    "#{root_path}cron/namespaces/#{route_params[:namespace]}"
  end
end
```
Or would it be better to create a separate helper file?

Reply:
I think it's fine for now keep it inline. We can create a separate module in the future when more helpers are implemented.